docs/prometheus-integration.md

```diff
@@ -191,6 +191,19 @@ internal-networkpolicy processed
 - **antrea_controller_network_policy_sync_duration_milliseconds:** The
 duration of syncing internal-networkpolicy
 
+#### Antrea Proxy Metrics
+
+- **antrea_proxy_total_endpoints_installed:** The number of Services installed
```

Comment:
typo: s/The number of Services/The number of Endpoints

BTW instead of "at Antrea Proxy", I would recommend "by Antrea Proxy"

also don't be typically spell it "AntreaProxy"?

Reply:
Updated, also changed to `AntreaProxy`.